end_users/README.md

```diff
@@ -1,3 +1,24 @@
 # End Users
 
-Documentation coming soon...
+## Table of Contents
+
+* [Merge Options](#merge-options)
+  * [Setting the Correct Merge Option](#setting-the-correct-merge-option)
+
+## Merge Options
+
+Repositories that use BonnyCI should not use any merge strategy which changes the SHA of the commit. This is because [zuul](https://github.com/openstack-infra/zuul), the scheduler used in BonnyCI, will be thrown off by the change in SHA if multiple changes are running in the gate. Since the gate queue tests multiple changes together in their theoretical merge order, the change in SHA for one commit will confuse zuul when it begins to test commits that rely on the commit that has been changed.
```

Comment:
"BonnyCI tests the changes in the pull requests without altering them. To ensure that which is tested is what is merged, do not enable a merge strategy that would alter the pull request content upon merging it."

Something like that?